pkg/R/RDD.R

```diff
@@ -1844,3 +1844,53 @@ setMethod("cogroup",
             cogroup.rdd <- mapValues(groupByKey(union.rdd, numPartitions), 
                                      group.func)
           })
+
+# TODO:
+# bind to env?
```

Comment:
We could do that, but I am not sure `name` will be that frequently used. I am fine with leaving it as a TODO -- You could word it as something like `TODO: Consider caching the name in the RDD's environment`